### PR TITLE
fix: adjust send request action

### PR DESCRIPTION
**Pull request by uziasferreirazup**
## Notes
Adjust send request action to response a object with a json

### CODE REVIEW

File: backend/sample/core/src/main/kotlin/br/com/zup/beagle/sample/builder/ButtonScreenBuilder.kt [hunk 1]
```diff
@@ -37,6 +40,9 @@ import br.com.zup.beagle.widget.layout.Screen
 import br.com.zup.beagle.widget.layout.ScreenBuilder
 import br.com.zup.beagle.widget.ui.Button
 import br.com.zup.beagle.widget.ui.ImagePath.Local
+import br.com.zup.beagle.widget.ui.Text
+
+data class Address(val street: String)
 
 object ButtonScreenBuilder : ScreenBuilder {
     override fun build() = Screen(
```
Comment (botbeaglezup) [inline backend/sample/core/src/main/kotlin/br/com/zup/beagle/sample/builder/ButtonScreenBuilder.kt:48]: <table data-meta="generated_by_backend">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/ZupIT/beagle/blob/1a9d375ffb78aa0682ef917962aa02bffc01fd54/backend/sample/core/src/main/kotlin/br/com/zup/beagle/sample/builder/ButtonScreenBuilder.kt#L48"></span>The function build is too long (63). The maximum length is 60.</td>
    </tr>
  </tbody>
</table>

Comment (botbeaglezup) [inline backend/sample/core/src/main/kotlin/br/com/zup/beagle/sample/builder/ButtonScreenBuilder.kt:48]: <table data-meta="generated_by_android">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/ZupIT/beagle/blob/1a9d375ffb78aa0682ef917962aa02bffc01fd54/backend/sample/core/src/main/kotlin/br/com/zup/beagle/sample/builder/ButtonScreenBuilder.kt#L48"></span>The function build is too long (63). The maximum length is 60.</td>
    </tr>
  </tbody>
</table>

File: backend/sample/core/src/main/kotlin/br/com/zup/beagle/sample/builder/ButtonScreenBuilder.kt [hunk 2]
```diff
@@ -57,7 +63,26 @@ object ButtonScreenBuilder : ScreenBuilder {
             )
         ),
         child = Container(
+            context = ContextData(
+                id = "context",
+                value = Address(street ="teste")
```
Comment (matheusribeirozup) [inline backend/sample/core/src/main/kotlin/br/com/zup/beagle/sample/builder/ButtonScreenBuilder.kt:68]: What do you think of putting the text in English, in this case: `Address(street ="test")`

File: android/beagle/src/main/java/br/com/zup/beagle/android/context/ContextActionExecutor.kt [hunk 3]
```diff
@@ -58,14 +59,14 @@ internal class ContextActionExecutor {
...
-    private fun parseToJSONObject(value: Any): JSONObject {
+    private fun parseToJSONObject(value: Any): Any {
         return if (value is String || value is Number || value is Boolean) {
             JSONObject().apply {
                 put(DEFAULT_KEY_NAME, value)
             }
         } else {
-            val json = BeagleMoshi.moshi.adapter<Any>(value::class.java).toJson(value)
-            JSONObject(json)
+            if (value is Collection<*>) JSONArray(value) else
```
Comment (tuliopereirazup) [inline android/beagle/src/main/java/br/com/zup/beagle/android/context/ContextActionExecutor.kt:68]: Maybe breaking this if in two lines is better for understanding.

File: backend/sample/micronaut/src/main/resources/application.properties [hunk 4]
```diff
@@ -17,3 +17,4 @@
 jackson.serializationInclusion=NON_NULL
 jackson.serialization.indentOutput=true
 beagle.cache.endpoint.exclude=/image
+beagle.cache.enabled=false
```
Comment (tuliopereirazup) [inline backend/sample/micronaut/src/main/resources/application.properties:20]: I think its better to disable only on android platform as our backend should have this feature enabled by default.

File: backend/sample/spring/src/main/resources/application.properties [hunk 5]
```diff
@@ -16,4 +16,5 @@
 
 beagle.cache.endpoint.exclude=/image
 spring.jackson.default-property-inclusion=NON_NULL
-spring.jackson.serialization.indent-output=true
\ No newline at end of file
+spring.jackson.serialization.indent-output=true
```
Comment (tuliopereirazup) [inline backend/sample/spring/src/main/resources/application.properties:19]: I think its better to disable only on android platform as our backend should have this feature enabled by default.

**Pull request merged**